ambuda/filters.py

```diff
@@ -7,8 +7,15 @@
 from markdown_it import MarkdownIt
 
 
-# Markdown parser.
-# Docs: https://markdown-it-py.readthedocs.io/en/latest/using.html
+#: A markdown parser for user-generated text.
+#:
+#: - `js-default` is like Commonmark but it disables raw HTML.
+#: - `typographer` enables specific typography improvements:
+#:   - `replacements` replaces `---` with `&mdash;`, etc.
+#:   - `smartquotes` replaces basic quotes with opening and closing quotes.
+#:   - `linkify` converts URLs like `"github.com"` into clickable links.
+#:
+#: Docs: https://markdown-it-py.readthedocs.io/en/latest/using.html
 MARKDOWN = MarkdownIt("js-default", {"typographer": True, "linkify": True}).enable(
```

Comment:
Should we link to the user guide from https://ambuda.org/proofing/beginners-guide?

Reply:
It seems quite technical and I'm happy to leave it in the Help button for now. Perhaps we can do this as part of a larger refactor that also takes care of #157.